### PR TITLE
WIP - Don't show edit button on research note to moderators - Issue: #5047

### LINKED ISSUE #5047: Don't show edit button on research note to moderators
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** this line https://github.com/publiclab/plots2/blob/master/app/views/like/_like.html.erb#L2

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
-  <% if logged_in_as(['admin', 'moderator']) || (current_user && (current_user.uid == node.uid || node.has_tag("with:#{current_user.username}"))) %>


+  <% if logged_in_as(['admin'']) || (current_user && (current_user.uid == node.uid || node.has_tag("with:#{current_user.username}"))) %>

```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by kaylawoodbury**
Fixes #5047

## Changes Made:
Updates this line https://github.com/publiclab/plots2/blob/master/app/views/like/_like.html.erb#L2 
**to**  <% if logged_in_as(['admin'']) ||.....  
**from** <% if logged_in_as(['admin', 'moderator']) || .....

_note: changes made in co-commit between myself and @JaimeCrz_

plots2_no_button

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/like/_like.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="like-buttons d-print-none btn-toolbar" style="margin-bottom :4px;">
-    <% if logged_in_as(['admin', 'moderator']) || (current_user && (current_user.uid == node.uid || node.has_tag("with:#{current_user.username}"))) %>
+    <% if logged_in_as(['admin']) || (current_user && (current_user.uid == node.uid || node.has_tag("with:#{current_user.username}"))) %>
         <% if node.type == 'note' %>
             <a aria-label="Edit post" <% if current_user && current_user.uid != node.uid %>data-confirm='Please be aware that you are editing a post by another contributor, a privilege you have only because you are an admin or co-author.' <% end %> class='btn btn-outline-secondary btn-circle btn-sm' href='/notes/edit/<%= node.id %>?t=<%= Time.now.to_i %><% if params[:controller] == "questions" %>&redirect=question&template=question<% end %>'>
                 <i class='ml-1 ff fa fa-pencil'></i>
```
Comment (TildaDares) [inline app/views/like/_like.html.erb:5]: @kaylawoodbury Awesome work but the only file change that needs to be in this PR is this one. The other changes are not needed.

**Review by TildaDares — CHANGES_REQUESTED** [review]
@kaylawoodbury Awesome work but the only file change that needs to be in this PR is

**Comment by cesswairimu** [comment]
hi @kaylawoodbury, just checking if you need  help in any way here. thanks

File: app/views/like/_like.html.erb (hunk 1 above)
Comment (kaylawoodbury) [reply app/views/like/_like.html.erb:5]: Thanks! I'll remove the changes in the other files, I think they happened when I was setting up to run everything locally.

**Comment by kaylawoodbury** [comment]
@cesswairimu Yes, sorry about the delay. Got busier than I was expecting the past couple of weeks. Will make the requested change today to wrap up this PR, thanks for checking in!

**Comment by cesswairimu** [comment]
no worries, thanks

**Comment by noi5e** [comment]
Hi @kaylawoodbury, let me know if you need any help finishing this one up!

**Comment by Tlazypanda** [comment]
Hmm... looks like this issue is already solved :sweat_smile: let me see if I can find another fto for you @kaylawoodbury

**Pull request closed without merging**